zappend/tailoring.py

```diff
@@ -40,9 +40,18 @@ def tailor_slice_dataset(
         # as dimension, e.g., "x", "y", "crs", ...
         dataset = dataset.drop_vars(const_variables)
 
+    # https://github.com/bcdev/zappend/issues/56
+    # slice_dataset.to_zarr(store, mode="a", ...) will replace
+    # global attributes.
+    # Therefore, we must replace slice dataset attributes by
+    # existing target dataset attributes.
+    # However, users should be able to select the appropriate
```

Comment:
Better create a new issue and refer to it instead of describing this in a comment.

Reply:
This has already been done, see #59